### ISSUE TITLE
cuhk03's protocol

### ISSUE THREAD

**Issue by liangbh6** [huanghoujing/AlignedReID-Re-Production-Pytorch#7]
@huanghoujing Hi, I reallly appreciate your work and want to use it to get the paper'result on cuhk03, can you help descript how to modify your code to get this result?

**Comment by huanghoujing** [comment]
In fact, there is no standard train/test split for CUHK03 in recent papers.

1. If you want to use the new train/test protocol as I provide here (which is popular in recent papers because it provides much larger test set), you only have to set the `dataset` argument to `cuhk03` when running the script `script/tri_loss/train.py`.

2. If you want to use the train/test protocol in the paper. At the end of Section 4.1 of the AlignedReID paper, it says
    >We only randomly split the dataset once for training and testing, and the gallery includes 200 identities.

    So when transforming CUHK03 using `script/dataset/transform_cuhk03.py`, you only have to provide your own `train_test_partition_file` corresponding to this split. This `train_test_partition_file` specifies image names for `train`, `query` and `gallery` sets.

**Comment by liangbh6** [comment]
@huanghoujing Nice explanation!  Thanks.
